src/screens/navscreens/MyPickupScreen.js

```diff
@@ -1,12 +1,123 @@
 import React from 'react';
-import {StyleSheet, View, Text} from 'react-native';
+import { StyleSheet, View, Text, Image } from 'react-native';
 import NavScreenHeader from '../../components/general/NavScreenHeader';
+import { NavigationActions } from 'react-navigation';
+import { FlatList } from 'react-native-gesture-handler';
 
+const courierImgs = {
+  ups: 'https://cdn.iconscout.com/icon/free/png-512/ups-282281.png',
+  usps: 'https://cdn4.iconfinder.com/data/icons/logos-and-brands/512/353_Usps_logo-512.png',
+  fedex: 'https://cdn.iconscout.com/icon/free/png-512/fedex-3-283136.png',
+};
+
+const pickups = [
+  {
+    id: 1,
+    key: 'Asus Laptop',
+    date: 'Dec 17, 2019 2PM - 4PM',
+    status: 'Awaiting pickup',
+    courier: 'usps',
+    delivery: 1,
+    price: 4.00,
+    printing: 0.5,
+    packaging: 2.00,
+    tax: 0.75
+  },
+  {
+    id: 2,
+    key: 'Macbook Pro',
+    date: 'Dec 18, 2019 2PM - 4PM',
+    status: 'In transit',
+    courier: 'ups',
+    delivery: 1,
+    price: 4.00,
+    printing: 0.5,
+    packaging: 2.00,
+    tax: 0.75
+  },
+  {
+    id: 3,
+    key: 'Surfacebook',
+    date: 'Dec 19, 2019 2PM - 4PM',
+    status: 'Completed',
+    courier: 'fedex',
+    delivery: 1,
+    price: 4.00,
+    printing: 0.5,
+    packaging: 2.00,
+    tax: 0.75
+  },
+];
+
+const styles = StyleSheet.create({
```

Comment:
Can you put styles below the class? Not a big deal but since I've put it below for other classes, just going for consistency. ty